in ARM64:

```
clang -c -o dlls/kernel32/thread.o ../dlls/kernel32/thread.c -Idlls/kernel32 -I../dlls/kernel32 -Iinclude -I../include \                                          -I../include/msvcrt -D__WINESRC__ -D_KERNEL32_ -D_NORMALIZE_ -D_MSVCR_VER=0 -D_REENTRANT -fPIC \                                                                -fasynchronous-unwind-tables -fno-builtin -fshort-wchar -Wall -pipe -fcf-protection=none \                                                                      -fno-stack-protector -fno-strict-aliasing -Wdeclaration-after-statement -Wempty-body \              
  -Wignored-qualifiers -Winit-self -Wno-pragma-pack -Wstrict-prototypes -Wtype-limits -Wvla \                                                                     -Wwrite-strings -Wpointer-arith -D_GLIBCXX_ASSERTIONS -D_FORTIFY_SOURCE=2 -pipe -Wno-error  -fstack-protector-strong --param=ssp-buffer-size=4  -fexceptions  -fPIC  -fPIE  -ggdb    -O2    -march=armv8-a -mtune=cortex-a53                  -fPIC -ffixed-x18 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0                         ../dlls/hidclass.sys/pnp.c:261:9: error: void function 'create_child' should not return a value [-Wreturn-type]
 ```

--- a/extra-emulation/wine/autobuild/amd64/build
+++ b/extra-emulation/wine/autobuild/amd64/build
@@ -1,3 +1,7 @@
+abinfo "Applying Wine Staging patches ..."
+ln -sv "$SRCDIR" "$SRCDIR"/../wine-staging/patches/wine
+"$SRCDIR"/../wine-staging/patches/patchinstall.sh DESTDIR="$SRCDIR" --all
+
 export CC="gcc-multilib-wrapper" CXX="g++-multilib-wrapper"
 
 abinfo "Building Wine (64-bit runtime) ..."

--- a/extra-emulation/wine/autobuild/prepare
+++ b/extra-emulation/wine/autobuild/prepare
@@ -1,7 +1,3 @@
-abinfo "Applying Wine Staging patches ..."
-ln -sv "$SRCDIR" "$SRCDIR"/../wine-staging/patches/wine
-"$SRCDIR"/../wine-staging/patches/patchinstall.sh DESTDIR="$SRCDIR" --all
-
 abinfo "Setting -fPIC ..."
 export CFLAGS="${CFLAGS} -fPIC"
 export LDFLAGS="${LDFLAGS} -fPIC"

